src/main/java/com/zendesk/maxwell/MaxwellContext.java

```diff
@@ -76,8 +81,11 @@ public Connection getRawMaxwellConnection() throws SQLException {
 	}
 
 	public void start() {
-		SchemaScavenger s = new SchemaScavenger(this.maxwellConnectionPool);
```

Comment:
nah, we pretty much keep schemas for all time now.  need to tear down the class too I htink.